reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -191,7 +191,8 @@ def copy_checkpoints_to_model_output(self):
                     if filename.startswith("checkpoint"):
                         checkpoints.append(os.path.join(root, filename))
             time.sleep(5)
-            if count >= 6:
+            # modify 'count' to 10 to be more reliable
+            if count >= 10:
```

Comment:
Just curious, was there a specific reason to increase this? Logically speaking the experiment trial already should already have finished when this method is called. If it was for the checkpoint sync down error, adding more counts would solve the issue.

Reply:
not sure if this can mitigate the ckpt issues, but no harm for extra 20s waiting time

Reply:
I doubt if this will make any difference in terms of locating the checkpoint. 30s seems a long wait time already.